src/Symfony/Component/HttpClient/composer.json

```diff
@@ -25,7 +25,7 @@
         "php": ">=7.2.5",
         "psr/log": "^1|^2|^3",
         "symfony/deprecation-contracts": "^2.1|^3",
-        "symfony/http-client-contracts": "^2.4",
+        "symfony/http-client-contracts": "^2.6",
```

Comment:
this should be updated to the next 2.4 I guess?

Reply:
see symfony/symfony@a95835f07f904fafdc8b722c1361ef438b73c12a